Flow.Launcher.Infrastructure/UserSettings/Settings.cs

```diff
@@ -177,6 +177,13 @@ public string QuerySearchPrecisionString
 
         public ObservableCollection<CustomPluginHotkey> CustomPluginHotkeys { get; set; } = new ObservableCollection<CustomPluginHotkey>();
 
+        public ObservableCollection<CustomShortcutModel> CustomShortcuts { get; set; } = new ObservableCollection<CustomShortcutModel>();
+
+        [JsonIgnore]
+        public ObservableCollection<BuiltinShortcutModel> BuiltinShortcuts { get; set; } = new ObservableCollection<BuiltinShortcutModel>() { 
+            new BuiltinShortcutModel("{clipboard}", "Get text from clipboard.", Clipboard.GetText)  // TODO: translation?
```

Comment:
How to translate this string?

Reply:
Try `Internationalization.Instance.GetTranslation(string key)`? I am not sure whether it is available when Settings is being constructed, but probably should?

Reply:
It's not. `Internationalization.Instance` initializes after settings because it needs to know selected language.

Reply:
You cannot use translation in this class and infrastructure space in general because it is instantiated late and has a dependency on settings class.

Reply:
Translation can be done when calling `get` method.

Reply:
Are we placing this in the get method in this pr?

Reply:
Maybe instead of the actual string, put the key in, and in the get call do the translation of the key.

Reply:
> Are we placing this in the get method in this pr?

If we decide to include built-in shortcuts in this pr I will implement this translation.